### ISSUE TITLE
[Lastname] [Firstname]

### ISSUE THREAD

**Issue by yozhi** [YetiForceCompany/YetiForceCRM#2351]
![vtiger](https://cloud.githubusercontent.com/assets/16100495/11500997/c09ca33e-9828-11e5-86b2-20eac1a7862c.png)
hello,
How can i change the display of full name of users to [Firstname] [Lastname] ? it displays too on "assigned to"and everything else, how can i change it ? 

regards

**Comment by eptapsyxos** [comment]
I suppose through language managment find the language you use edit language find the module you refer Android Change  it.

**Comment by yozhi** [comment]
i didn't find it in language managment, any more idea how i can change it ?

**Comment by yozhi** [comment]
if someone got this like i want , could upload your Yetiforce files for the followings paths: models/users;
models/customview ; models/vtiger ; include/utils. i would be grateful , and if you got the developer version, it would be better

**Comment by eptapsyxos** [comment]
I have managed the first name..but the last name not yet

**Comment by yozhi** [comment]
where did you do that? i couldn't found it and its important for me to do that

**Comment by eptapsyxos** [comment]
The only thing i did was from translation. The Last name i also translate it but its doesnt want to be translated. Look at language managment for vtiger settings module. I Dont remember clearly but there Must to be. Ctrl+f And search for Last name.

**Comment by yozhi** [comment]
at language management we only can translate the labels name, not the way we see the labels , if you know what i mean

**Comment by eptapsyxos** [comment]
Ok sorry misunderstood. I can not help you on this. Hope  to find  your  answers.

**Comment by yozhi** [comment]
no problem, thank you anyway

**Comment by bpabiszczak** [comment]
Check this table: vtiger_entityname. Be careful what you do!

**Comment by yozhi** [comment]
dont worry, but it resulted, thank you very much, i was searching ,but i forgot to search on mysql haha one more time, thank you very much

**Comment by nikdejan** [comment]
In vtlib/Vtiger/Functions.php after line 471 add this two lines:
		if ($module == 'Users') {
			$label_name=array_reverse($label_name);
		}
This has not been tested and I don't know what are all the consequences of this hack, so do it on your own responsibility.

**Comment by bpabiszczak** [comment]
@nikdejan Do you like to spoil the system? ;)

**Comment by nikdejan** [comment]
Well, maybe... sometimes ;-)

**Comment by nikdejan** [comment]
And if I don't spoil the system who is going to pay you to fix it? ;-)

**Comment by yozhi** [comment]
the easy was to change on database, its easier

**Comment by yozhi** [comment]
i spent many hours trying to change the system and was a waste of time

**Comment by yozhi** [comment]
i got this problem , changing on vtiger_entityname fixed the displaying users name , but on search column i got this problem
![animation](https://cloud.githubusercontent.com/assets/16100495/12350516/696e7276-bb6f-11e5-96da-f83edb2ed157.gif)

**Comment by luminouscomm** [comment]
I followed yozhi's advice and made the change at the DB level. I had to go through phpMyAdmin to make the changes. Here's a detailed list of what I did:
Found my Yeti Database
Went in to the vtiger_entityname table
Edit "Users" under the "modulename" column
Both the **fieldname** and the **searchcolumn** values were set to last_name,first_name
I changed both of these fields to **first_name,last_name** and this seems to have fixed my issue. 

I hope this helps other users.
